_posts/2015/2016-07-20-fullstackcon-2016-retrospective.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages goes FullStack JavaScript conference"
+date: "2016-07-20 12:00:00"
+image: blog-header/javascript.png
```

Comment:
I think [you added that image originally](https://github.com/epages-de/epages-docs/commit/1cda6a227284bf759966e1f2b714336794525fb0).